annotator/__main__.py

```diff
@@ -102,12 +103,13 @@ def json2table(args, syn):
         all_modules.append(module_df)
 
     # concat the list of all normalized dataframes into one annotation dataframe
-    all_modules_df = pandas.concat(all_modules)
+    all_modules_df = pandas.concat(all_modules, sort = False)
 
     # re-arrange columns/fields and sort data.
     all_modules_df = all_modules_df[annotation_schema]
     all_modules_df.sort_values(key, ascending=[True, True, True], inplace=True)
-    all_modules_df.valueDescription = all_modules_df.valueDescription.str.encode('utf-8')
+    # no need for line below on python3/pandas > 20.0
+    #all_modules_df.valueDescription = all_modules_df.valueDescription.str.encode('utf-8')
```

Comment:
Should we leave this in but make it conditional on the python/pandas version, or are we officially dropping support for python 2 in this code? I don't know who else uses this, so that might require a wider discussion.

Reply:
Yes - going back to the previous discussion about package maintenance: https://github.com/Sage-Bionetworks/annotator/pull/90#discussion_r254396535

Agreed if we are maintaining the package, we can explicitly state which versions of pandas/python we support, if we don't provide backward compatibility (which would be hard in the case of python 2 to python 3 related changes).